Rust std::fmt::Display doesn't expect a C void*, change to print it in
hexadecimal

--- third_party/rust/audio_thread_priority/src/rt_linux.rs.orig	2019-10-22 12:20:33.250687971 +0700
+++ third_party/rust/audio_thread_priority/src/rt_linux.rs	2019-10-22 12:39:10.400928328 +0700
@@ -100,7 +100,7 @@
     let mut policy = 0;
 
     if unsafe { libc::pthread_getschedparam(pthread_id, &mut policy, &mut param) } < 0 {
-        error!("pthread_getschedparam error {}", pthread_id);
+        error!("pthread_getschedparam error {:?}", pthread_id);
         return Err(());
     }
 
@@ -127,7 +127,7 @@
     if unsafe { libc::pthread_setschedparam(rt_priority_handle.pthread_id,
                                             rt_priority_handle.policy,
                                             &rt_priority_handle.param) } < 0 {
-        warn!("could not demote thread {}", rt_priority_handle.pthread_id);
+        warn!("could not demote thread {:?}", rt_priority_handle.pthread_id);
         return Err(());
     }
     return Ok(());
